package.json

```diff
@@ -69,7 +69,7 @@
         "@duckduckgo/jsbloom": "^1.0.2",
         "@duckduckgo/privacy-dashboard": "github:duckduckgo/privacy-dashboard#1.6.1",
         "@duckduckgo/privacy-grade": "file:packages/privacy-grade",
-        "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#main",
+        "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#gd-hostname-ta-test",
```

Comment:
This will be merged first right?

Reply:
We have a deadlock - https://github.com/duckduckgo/privacy-reference-tests/pull/108 is blocked waiting for BSK, but if I point the extension to the reference test main branch the tests fail, as we're missing some MV3 exceptions. I'd suggest we just point at this branch temporarily until that PR gets merged.